tests/integration/test_graphite_merge_tree/test.py

```diff
@@ -234,18 +234,20 @@ def test_paths_not_matching_any_pattern(graphite_table):
 
 def test_system_graphite_retentions(graphite_table):
     expected = '''
-graphite_rollup	\\\\.count$	sum	0	0	1	0	['test']	['graphite']
-graphite_rollup	\\\\.max$	max	0	0	2	0	['test']	['graphite']
-graphite_rollup	^five_min\\\\.		31536000	14400	3	0	['test']	['graphite']
-graphite_rollup	^five_min\\\\.		5184000	3600	3	0	['test']	['graphite']
-graphite_rollup	^five_min\\\\.		0	300	3	0	['test']	['graphite']
-graphite_rollup	^one_min	avg	31536000	600	4	0	['test']	['graphite']
-graphite_rollup	^one_min	avg	7776000	300	4	0	['test']	['graphite']
-graphite_rollup	^one_min	avg	0	60	4	0	['test']	['graphite']
+graphite_rollup	all	\\\\.count$	sum	0	0	1	0	['test']	['graphite']
+graphite_rollup	all	\\\\.max$	max	0	0	2	0	['test']	['graphite']
+graphite_rollup	all	^five_min\\\\.		31536000	14400	3	0	['test']	['graphite']
+graphite_rollup	all	^five_min\\\\.		5184000	3600	3	0	['test']	['graphite']
+graphite_rollup	all	^five_min\\\\.		0	300	3	0	['test']	['graphite']
```

Comment:
Let's add some testing types for plain and tagged rules?

Reply:
May be, but for result of this need add several rollup rules files (current and additional - with separated plain/tagged rules)